src/index.js

```diff
@@ -57,7 +57,7 @@ import jsonFields from './json-fields';
  *
  * See methods below for details.
  */
-export default (Bookshelf, options = {}) => {
+export default (Bookshelf, options = { nullString: 'null' }) => {
```

Comment:
If an options object is provided without `nullString`, then `nullString` will not be defined, thus breaking the functionality. The default should not be defined like this.